modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -116,6 +116,12 @@ rhn-ssl-tool --gen-ca --rpm-only --dir="/root/ssl-build" --from-ca-cert=<Path_to
 rhn-ssl-tool --gen-server --rpm-only --dir="/root/ssl-build" --from-server-key=<Server_Key_File> --from-server-cert=<Server_Cert_File>
 ----
 
+. Install the new certificate:
++
+----
+cp new-server.key /etc/apache2/ssl.key/server.key
+----
```

Comment:
We generated an RPM so we need to install it. We described it in "ssl-certs-selfsigned". So maybe something like this?

```suggestion
. Install the RPM that contains the newly generated certificate.
    Check that you have the latest version of the RPM before running this command.
    The version number is incremented every time you re-create the certificates.
+
----
rpm -Uhv /root/ssl-build/sumahostname/rhn-org-httpd-ssl-key-pair-sumahostname-1.0-2.noarch.rpm
----
. Restart services to pick up the changes:
+
----
spacewalk-service restart
----
```